docs/src/main/sphinx/connector/hive-s3.rst

```diff
@@ -310,6 +310,8 @@ Property name                         Description
 
```

Comment:
Regarding the commit message, please follow 

https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#commits-and-pull-requests

(trim the lines of the commit content to 70/80, remove the reference to the initial issue - it can be retrieved from the PR)